Add RememberingEntry

mikitsu opened the pull request:
Adds a class for Entry widgets remembering previous inputs

mikitsu committed:
Add RemembringEntry

Adds a class for Entry widgets remembering previous inputs

pep8speaks commented:
Hello @mik2k2! Thanks for opening this PR. 

* In the file [`misc/tkstuff/__init__.py`](https://github.com/mik2k2/misc-utils/blob/cd89874e2737144f4e1dd2cde343fc903cbdc235/misc/tkstuff/__init__.py):

> [Line 523:1](https://github.com/mik2k2/misc-utils/blob/cd89874e2737144f4e1dd2cde343fc903cbdc235/misc/tkstuff/__init__.py#L523): [W293](https://duckduckgo.com/?q=pep8%20W293) blank line contains whitespace
> [Line 527:1](https://github.com/mik2k2/misc-utils/blob/cd89874e2737144f4e1dd2cde343fc903cbdc235/misc/tkstuff/__init__.py#L527): [W293](https://duckduckgo.com/?q=pep8%20W293) blank line contains whitespace
> [Line 530:1](https://github.com/mik2k2/misc-utils/blob/cd89874e2737144f4e1dd2cde343fc903cbdc235/misc/tkstuff/__init__.py#L530): [W293](https://duckduckgo.com/?q=pep8%20W293) blank line contains whitespace
> [Line 535:5](https://github.com/mik2k2/misc-utils/blob/cd89874e2737144f4e1dd2cde343fc903cbdc235/misc/tkstuff/__init__.py#L535): [E301](https://duckduckgo.com/?q=pep8%20E301) expected 1 blank line, found 0
> [Line 538:1](https://github.com/mik2k2/misc-utils/blob/cd89874e2737144f4e1dd2cde343fc903cbdc235/misc/tkstuff/__init__.py#L538): [W293](https://duckduckgo.com/?q=pep8%20W293) blank line contains whitespace
> [Line 550:1](https://github.com/mik2k2/misc-utils/blob/cd89874e2737144f4e1dd2cde343fc903cbdc235/misc/tkstuff/__init__.py#L550): [W293](https://duckduckgo.com/?q=pep8%20W293) blank line contains whitespace
> [Line 555:1](https://github.com/mik2k2/misc-utils/blob/cd89874e2737144f4e1dd2cde343fc903cbdc235/misc/tkstuff/__init__.py#L555): [W293](https://duckduckgo.com/?q=pep8%20W293) blank line contains whitespace
> [Line 557:49](https://github.com/mik2k2/misc-utils/blob/cd89874e2737144f4e1dd2cde343fc903cbdc235/misc/tkstuff/__init__.py#L557): [E226](https://duckduckgo.com/?q=pep8%20E226) missing whitespace around arithmetic operator
> [Line 560:1](https://github.com/mik2k2/misc-utils/blob/cd89874e2737144f4e1dd2cde343fc903cbdc235/misc/tkstuff/__init__.py#L560): [W293](https://duckduckgo.com/?q=pep8%20W293) blank line contains whitespace
> [Line 567:1](https://github.com/mik2k2/misc-utils/blob/cd89874e2737144f4e1dd2cde343fc903cbdc235/misc/tkstuff/__init__.py#L567): [W293](https://duckduckgo.com/?q=pep8%20W293) blank line contains whitespace

Do see the [Hitchhiker's guide to code style](https://goo.gl/hqbW4r)

mikitsu committed:
Add change to VarWidget

The keyword argument for Entry widgets is `textvariable`; add support for choosing the kw-arg name

mik2k2 commented:
@pep8speaks pep8ify

pep8speaks committed:
Fix PEP 8 errors in misc/tkstuff/__init__.py

pep8speaks commented:
Here you go with [the Pull Request](https://github.com/mik2k2/misc-utils/pull/33) ! The fixes are suggested by [autopep8](https://github.com/hhatto/autopep8).

 @mik2k2

mikitsu committed:
Merge pull request #33 from pep8speaks/add-remeberingentry-pep8-patch

Fix PEP 8 errors

mikitsu committed:
reformat docstrings
